core/docs/spark-qualification-tool.md

```diff
@@ -981,14 +981,14 @@ Sample output in text:
 Note: This step is optional.
 
 ```bash
-git clone https://github.com/NVIDIA/spark-rapids.git
-cd spark-rapids
-mvn -Pdefault -pl .,tools clean verify -DskipTests
+git clone https://github.com/NVIDIA/spark-rapids-tools.git
+cd spark-rapids-tools/core
+mvn -Pdefault clean verify -DskipTests
```

Comment:
```suggestion
mvn clean package -DskipTests
```

Reply:
Ignoring since I plan to remove the steps in the qualification tool doc and link to the spark-rapids-tools build link